tests/UI/pages/BO/international/currencies/add.ts

```diff
@@ -51,6 +74,23 @@ class AddCurrency extends LocalizationBasePage {
 
     // currency modal
     this.currencyLoadingModal = '#currency_loading_data_modal';
+
+    // Currency Format
+    this.currencyFormatDiv = '#currency_formatter';
+    this.currencyFormatTable = `${this.currencyFormatDiv} table`;
+    this.currencyFormatRows = `${this.currencyFormatTable} tbody tr`;
+    this.currencyFormatCell = (nthRow: number, nthColumn:number) => `${this.currencyFormatRows}:nth-child(${nthRow}) `
+      + `td:nth-child(${nthColumn})`;
+    this.currencyFormatRow = (nthRow: number) => `${this.currencyFormatRows}:nth-child(${nthRow})`;
```

Comment:
You can use thi selector to create `this.currencyFormatCell` thel use cell to create edit and reset selectors

Reply:
- rows
- row
- cell
- specific cell